robot_descriptions/skydio_x2_description.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2024 Stéphane Caron
```

Comment:
Feel free to put your copyright here (that's you or your employer if you do this at work), or remove the copyright line (it is not mandatory, the copyright exists whether it's written or not).